blockly-core/core/ui/block_space/horizontal_flyout.js

```diff
@@ -32,6 +32,18 @@ Blockly.HorizontalFlyout.prototype.show = function(xmlList) {
   Blockly.HorizontalFlyout.superClass_.show.apply(this, arguments);
 };
 
+Blockly.HorizontalFlyout.prototype.setVisibility = function(show) {
+  return show ? this.softShow() : this.softHide()
+};
```

Comment:
That's right, removing `return`, doesn't do anything.

I'll also add a comment on that, soft is to distinguish from the show/hide methods below it which are really `emptyAndHide` and `populateAndShow` methods. If the refactor looks quick enough I may just rename those to be accurate